modules/olm-enabling-operator-for-multi-arch.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-generating-csvs.adoc
+
+[id="olm-enabling-operator-for-multi-arch_{context}"]
+= Enabling your Operator for multiple architectures and operating systems
+
+Operator Lifecycle Manager (OLM) assumes that all Operators run on Linux hosts.
+However, as an Operator author, you can specify whether your Operator supports
+managing workloads on other architectures, if worker nodes are available in the
+{product-title} cluster.
+
+If your Operator supports variants other than AMD64 and Linux, you can add
+labels to the CSV that provides the Operator to list the supported variants.
+Labels indicating supported architectures and operating systems are defined by
+the following:
+
+[source,yaml]
+----
+labels:
+    operatorframework.io/arch.<arch>: supported <1>
+    operatorframework.io/os.<os>: supported <2>
+----
+<1> Set `<arch>` to a supported string.
+<2> Set `<os>` to a supported string.
+
+.Architectures supported on {product-title} {product-version}
+[options="header"]
+|===
+|Architecture |String
+
+|AMD64
+|`amd64`
+|===
+
+////
+Supported in 4.3 but not yet 4.4,
```

Comment:
If it is supported in 4.3 and the operator is to be available in the 4.3 then the head CSV of the default channel needs to state support. 

I would say that we should allow folks to say their operator supports an arch because of this and we should expose these values IMO

Reply:
OK, exposed them and dropped the version number from the table, but keeping the "NOTE" box that mentions different support per version of OCP/Kube-distros.